localstack/logging/format.py

```diff
@@ -0,0 +1,142 @@
+"""Tools for formatting localstack logs."""
+import logging
+from functools import lru_cache
+from typing import Dict
+
+from localstack.utils.numbers import format_bytes
+
+MAX_THREAD_NAME_LEN = 12
+MAX_NAME_LEN = 26
+
+LOG_FORMAT = f"%(asctime)s.%(msecs)03d %(ls_level)5s --- [%(ls_thread){MAX_THREAD_NAME_LEN}s] %(ls_name)-{MAX_NAME_LEN}s : %(message)s"
```

Comment:
nit: Great to have the thread information also available in the logs, although potentially we could think about adding that to the `trace`-enabled logs only. Just a thought for future iterations, let's see how things are evolving with having the thread info in all logs for now.. 👍